data/mods/Xedra_Evolved/effects/effects.json

```
@@ -505,7 +510,7 @@
     "type": "effect_type",
     "name": [ "Physical Instrument" ],
     "desc": [
-      "Your body is a tool of destruction.  Use it effectively.  Your attacks are faster and deadlier, and your dodge abilities is better"
+      "Your body is a tool of destruction.  <color_red>Use it effectively<color>.  Your attacks are faster and deadlier, and your dodge abilities is better"
```

```suggestion
      "Your body is a tool of destruction.  <color_red>Use it effectively<color>.  Your attacks are faster and deadlier, and your dodging ability is better."
```
(or "dodging abilities are better.")